components/breadcrumb/stories/breadcrumb.stories.js

```diff
@@ -228,6 +305,23 @@ Disabled.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+/**
+ * For the multiline variant, the breadcrumb title can be customized using an additional element that uses the heading classes from
```

Comment:
I think I'm a bit confused with this documentation. When I read this, particularly, "If no heading element or classes are used, the text will be sized the same as a large heading by default," I am expecting the default with no classes multiline to look more similar in font size to the multiline with the large title/heading. Is that true, or is that the intention? If so, I'm seeing the default no class multiline have only 18px in that heading, while the large title multiline has 28px. Maybe it wouldn't be the 800 font weight, but I am understanding that line as the text size will be 28px for both a large-sized title multiline, and a default multiline.

I might be misunderstanding, so feel free to tell me I'm wrong!

Reply:
I think something was going awry with the token versions after the recent rebase, I saw the wrong tokens loaded locally yesterday but not on the build link; I bumped up the required `next` version of tokens and it went away. Default multiline with no classes looks the same as Large with the heading.